src/google/protobuf/compiler/csharp/csharp_generator.cc

```diff
@@ -81,6 +81,13 @@ bool Generator::Generate(
     } else if (options[i].first == "base_namespace") {
       cli_options.base_namespace = options[i].second;
       cli_options.generate_directories = true;
+    } else if(options[i].first == "internal_access") {
+      std::string value = options[i].second;
+      if(value == "false" || value == "no" || value == "") {
```

Comment:
Makes sense (for parity). And also yes for ` (`

(I personally love true/false & yes/no, but that's me.)